docs/content/start/telemetry.md

```diff
@@ -0,0 +1,175 @@
+
+## ITelemetryBaseLogger interface
+
+The `ITelemetryBaseLogger` is an interface within the `@fluidframework/common-definitions` package. This interface can be implemented and passed into the `createContainer()` and `getContainer()` methods in `TinyliciousClient` as part of the `TinyliciousContainerConfig` type parameter.
+
+```ts
+export interface TinyliciousContainerConfig {
+  id: string;
+  logger?: ITelemetryBaseLogger;
+}
+```
+
+```ts
+public async createContainer(
+  serviceContainerConfig: TinyliciousContainerConfig,
+  containerSchema: ContainerSchema,
+): Promise<[container: FluidContainer, containerServices: TinyliciousContainerServices]>
+```
+
+
+### Properties and methods
+
+The interface contains a `supportTags` property and a `send()` method as shown:
+
+```ts
+export interface ITelemetryBaseLogger {
+  /**
+   * An optional boolean which indicates to the user of this interface that tags (i.e. `ITaggedTelemetryPropertyType`
+   * objects) are in use. Eventually this will be a required property, but this is a stopgap that allows older hosts
+   * to continue to pass through telemetry without trouble (this property will simply show up undefined), while our
+   * current logger implementation in `telmetry-utils` handles tags in a separate manner.
+   */
+  supportsTags?: true;
+  send(event: ITelemetryBaseEvent): void;
+}
+```
+
+- SupportsTags
+  - These tags are generic strings used to classify different events. In a simple logger, all events are untagged and handled the same by your logger's implementation. However, in some scenarios, where some data should be handled separately (e.g. private customer data), then it would be worthwhile to "tag" the event with some identifier.
+- `send()`
+  - The `send()` method is called by the container's telemetry system whenever a telemetry event occurs. This method takes in an ITelemetryBaseEvent type parameter, which is also within the `@fluidframework/common-definitions` package. Given this method is part of an interface, users can implement a custom telemetry logic for the container's telemetry system to execute.
+
+### Adding complexity
+
+Different levels of logging complexity can be achieved by adding other attributes to the object implementing the `ITelemetryBaseLogger` interface. For example, the `ITelemetryLogger` interface, also within the `@fluidframework/common-definitions` package, broke down telemetry events into different categories, allowing different logging logics for each category. However, it is imperative to ensure that the `send()` method is ultimately called since it is the actual method that is piped to the container's telemetry system and sends the telemetry event.
```

Comment:
```suggestion
Different levels of logging complexity can be achieved by adding other attributes to the object implementing the `ITelemetryBaseLogger` interface. For example, the `ITelemetryLogger` interface, also within the `@fluidframework/common-definitions` package, breaks down telemetry events into different categories, allowing different logging logic for each category. However, it is imperative to ensure that the `send()` method is ultimately called since it is the actual method that is piped to the container's telemetry system and sends the telemetry event.
```

Reply:
The point about `ITelemetryBaseLogger` v. other enrichment is that _between layers of the Fluid Framework_ only `ITelemetryBaseLogger` should be used.